larky/src/main/java/com/verygood/security/larky/parser/DefaultLarkyInterpreter.java

```diff
@@ -43,7 +43,7 @@ public DefaultLarkyInterpreter(StarlarkMode mode, Bindings... bindings) {
   }
 
   public ParsedStarFile evaluate(StarFile script, Writer writer) throws IOException, EvalException {
-    return evaluate(script, new StreamWriterConsole(writer));
+    return evaluate(script, new StreamWriterConsole(writer, true));
```

Comment:
As an alternative, could we add 
`evaluate(StarFile script, Writer writer, boolean includeLineNumbers)`
as a separate method? Would we be able to call that or would it require more plumbing?

Reply:
Or we can change the sig of this method

Reply:
Or we could add a flag to the bindings 🤔

Reply:
> As an alternative, could we add `evaluate(StarFile script, Writer writer, boolean includeLineNumbers)` as a separate method? Would we be able to call that or would it require more plumbing?

It will require changes in [LarkyCompiledScript](https://github.com/verygoodsecurity/starlarky/blob/6bc155ffd2aed4c20002638a98431d77beb5660c/larky/src/main/java/com/verygood/security/larky/jsr223/LarkyCompiledScript.java#L55)

> Or we can change the sig of this method

This would require a change in [Reporter](https://github.com/verygoodsecurity/starlarky/blob/6bc155ffd2aed4c20002638a98431d77beb5660c/larky/src/main/java/com/verygood/security/larky/modules/utils/Reporter.java#L31)

Reply:
```
    String verbose = interpreter.getGlobals().getOrDefault("__VERBOSE__", "false").toString();
    return evaluate(script, new StreamWriterConsole(writer, Boolean.parseBoolean(verbose)));
```

Reply:
Then add 
`bindings.put("__VERBOSE__", "true")`
in `ExecutionWrapper.constructBinding()`